server/bot_api.go

```diff
@@ -227,10 +227,12 @@ func (p *Plugin) handleBotPostRecordings(w http.ResponseWriter, r *http.Request)
 	}
 	post.AddProp("recording_files", recordingFiles)
 
+	T := p.getTranslationFunc("")
+
 	startAt, _ := post.GetProp("start_at").(int64)
-	postMsg := "Here's the call recording"
+	postMsg := T("app.call.new_recording_message")
```

Comment:
oh no... we have to name them? boo

Reply:
Yeah, although it shouldn't be too challenging to generate hash-based IDs I suppose but it'll require changes in https://github.com/mattermost/mattermost-utilities/tree/master/mmgotool